subsys/bluetooth/audio/mpl.c

```diff
@@ -261,14 +261,24 @@ enum mpl_objects {
 /* And, except for the icon object, all objects can be created dynamically. */
 /* So a single buffer to hold object content is sufficient.  */
 struct obj_t {
-	uint64_t selected_id;  /* ID of the currently selected object*/
-	bool     busy;
-	uint8_t  add_type;     /* Type of object being added, e.g. MPL_OBJ_ICON */
+	/* ID of the currently selected object*/
+	uint64_t selected_id;
+
+	bool busy;
+
+	/* Type of object being added, e.g. MPL_OBJ_ICON */
+	uint8_t add_type;
+
+	/* Parameters of object being added */
+	struct bt_ots_obj_created_param param;
```

Comment:
I might be tired, but could you briefly describe why we need to store this value here?

Reply:
I hadn't completed this yet and the compilation of this module is failing at the moment for that reason but eventually the created callback needs to use these parameters to provide the created object parameters to the OTS layer.

This could be a pointer if that is what you mean, and the actual storage can be on the stack of the caller which would save some space.

Reply:
I have updated this now and converted this to a pointer to save some space.

Reply:
Sorry for reviewing too quickly :)

Reply:
I appreciate your prompt and valuable comments 👍🏼